terraform-incubator/people-depot/dev/main.tf

```diff
@@ -9,7 +9,7 @@ terraform {
 }
 
 provider "aws" {
-  region = "us-west-2"
+  region = "us-east-2"
```

Comment:
curious about this change. this is to a main.tf file, not a readme...and the region should prob be us-west-2